docker-compose.link.yaml

```diff
@@ -13,7 +13,16 @@ services:
         npm run start:watch"
     networks:
       - ffc-demo
+    ports:
+      - "3003:3003"
```

Comment:
Ports duplicated from `docker-compose.override.yaml` to enable direct access to API when running with `--link`. Not moved back to base file as that would cause test runs to conflict with the running app.